app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.